changelog/next/features/4564--tcp-listen-metrics.md

```diff
@@ -0,0 +1,3 @@
+We added a metric for tcp connections. It emits one event per second for every
```

Comment:
The _it emits_ part threw me off when reading this. The metric doesn't emit events, instead pipelines using the TCP connector now emit metrics once per second and connection.